spec/liquid_tags/link_tag_spec.rb

```diff
@@ -74,6 +74,18 @@ def correct_link_html(article)
     expect(liquid.render).to eq(correct_link_html(article))
   end
 
+  it "renders default reading time of 1 minute for short articles" do
+    liquid = generate_new_liquid("/#{user.username}/#{article.slug}/")
+    expect(liquid.render).to include('1 min read')
+  end
+
+  it "renders reading time of article lengthy articles" do
+    template = file_fixture("article_long_content.txt").read
+    article = create(:article, user_id: user.id, title: "test this please", tags: "tag1 tag2 tag3", body_markdown: template)
```

Comment:
This can be simplified a bit.
```suggestion
  article = create(:article, user: user, body_markdown: template)
```

Reply:
Sure it can be. Thanks!